redaxo/src/addons/media_manager/pages/types.php

```diff
@@ -87,6 +129,22 @@
     $list->setColumnFormat('name', 'custom', function ($params) {
         $list = $params['list'];
         $name = '<b>' . $list->getValue('name') . '</b>';
+        $name .= '<div class="pull-right">
+            <button
+                onclick="rex_mediamanager_toggle(this)"
+                type="button"
+                class="btn btn btn-xs rex-mediamanager-list-effect-btn"
+                data-target=".rex-mediamanager-list-'.$list->getValue('name').'-effect .panel-body">
+                <i class="rex-icon  fa-search-plus"></i>
```

Comment:
für `fa-search-plus` in `_icon.scss` einen `rex-icon` Eintrag anlegen.

Reply:
in der `redaxo/src/addons/be_style/package.yml` gibt es die option `compile`. Wenn du die auf `1` stellst, wird der Compiler angeworfen.